pkgs/thycotic/dsv-cli/registry.yaml

```diff
@@ -0,0 +1,31 @@
+packages:
+  - type: github_release
+    repo_owner: thycotic
+    repo_name: dsv-cli
+    asset: dsv-{{.OS}}-{{.Arch}}
+    format: raw
+    replacements:
+      amd64: x64
+      arm64: x64
+      386: x86
+      arm64n: arm64
+      windows: win
+    supported_envs:
+      - darwin
+      - amd64
+    checksum:
```

Comment:
Please see the comment.
https://github.com/aquaproj/aqua-registry/pull/6892#issuecomment-1272165909

Reply:
Ok, if it's still wrong please let me know. I worked through this with the replacement naming specifically to match what's in the repo. If i need to go through the defaults instead and it works I can do that.